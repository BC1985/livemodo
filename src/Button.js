import React from "react";

const buttonStyle = {
  width: "100px",
  height: "28px",
  borderRadius: "5px",
  backgroundColor: "#373f51",
  color: "whitesmoke"
};
export default function Button(props) {
  return <button style={buttonStyle}>{props.children}</button>;
}
