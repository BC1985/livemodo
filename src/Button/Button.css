#button {
  border: 1px solid #202530;
  -webkit-border-radius: 5px;
  -moz-border-radius: 5px;
  border-radius: 5px;
  padding: 10px;
  text-decoration: none;
  display: inline-block;
  text-shadow: -1px -1px 0 rgba(0, 0, 0, 0.3);
  color: #ffffff;
  background-color: #373f51;
  background-image: -webkit-gradient(
    linear,
    left top,
    left bottom,
    from(#646d7e),
    to(#22242e)
  );

  cursor: pointer;
}

#button:hover {
  background-color: #222733;
  background-image: -webkit-gradient(
    linear,
    left top,
    left bottom,
    from(#373f51),
    to(#060c0f)
  );
}

/* ------ iPhone 6, 6S, 7, 8--------- */

/* Portrait */
@media only screen and (min-device-width: 375px) and (max-device-width: 667px) and (-webkit-min-device-pixel-ratio: 2) and (orientation: portrait) {
  #greeting {
    margin: 0 20% 0 20%;
    padding: 20px 0 20px 0;
  }
  #button {
    width: 100%;
    margin: 0px;
  }
}
