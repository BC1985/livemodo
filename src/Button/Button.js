import React from "react";
import "./Button.css";

export default function Button(props) {
  return <button id="button">{props.children}</button>;
}
