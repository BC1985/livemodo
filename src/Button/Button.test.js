import React from "react";
import ReactDOM from "react-dom";
import Button from "../Button/Button";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Button />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("renders UI as expected", () => {
  const tree = renderer.create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});
