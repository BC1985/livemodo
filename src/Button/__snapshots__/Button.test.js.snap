// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UI as expected 1`] = `
<button
  id="button"
/>
`;
